Assets/SEE/Controls/Actions/Drawable/MoveRotatorAction.cs

```diff
@@ -0,0 +1,864 @@
+                        oldObjectPosition, newObjectPosition, oldObjectLocalEulerAngles, degree, executedOperation,
+                        includeChildren);
+                    Destroyer.Destroy(selectedObject.GetComponent<Rigidbody>());
+                    Destroyer.Destroy(selectedObject.GetComponent<CollisionController>());
+                    GameMoveRotator.DestroyRigidBodysAndCollisionControllersOfChildren(selectedObject);
+                    new RbAndCCDestroyerNetAction(memento.drawable.ID, memento.drawable.ParentID,
+                        memento.selectedObject.name).Execute();
+                    RotationMenu.Disable();
+                    MoveMenu.Disable();
+                    CurrentState = IReversibleAction.Progress.Completed;
+                    return true;
+                } else
+                {
+                    if (executedOperation == ProgressState.Rotate)
+                    {
+                        /// A block is needed because occasionally a trigger exit is not registered during rotation. 
+                        /// This block attempts to set the isInCollision value of the Collision Controller to false. 
+                        /// However, if the object is still in a collision, it will be set back to true by its OnStayCollision method.
+                        selectedObject.GetComponent<CollisionController>().TrySetCollisionToFalse();
+                    }
+                }
+            }
+            else
+            {
+                /// Block for reset.
+                Destroyer.Destroy(selectedObject.GetComponent<Rigidbody>());
+                Destroyer.Destroy(selectedObject.GetComponent<CollisionController>());
+                GameMoveRotator.DestroyRigidBodysAndCollisionControllersOfChildren(selectedObject);
+                GameObject drawable = GameFinder.GetDrawable(selectedObject);
+                new RbAndCCDestroyerNetAction(drawable.name, GameFinder.GetDrawableParentName(drawable),
+                    selectedObject.name).Execute();
+                selectedObject = null;
+                RotationMenu.Disable();
+                MoveMenu.Disable();
+                progressState = ProgressState.SelectObject;
+            }
+            return false;
+        }
+
+        /// <summary>
+        /// Get the pressed KeyCode of the arrow keys.
+        /// </summary>
+        /// <returns>The pressed arrow key.</returns>
+        private KeyCode GetArrowKey()
+        {
+            if (Input.GetKey(KeyCode.LeftArrow))
+            {
+                return KeyCode.LeftArrow;
+            }
+            else if (Input.GetKey(KeyCode.RightArrow))
+            {
+                return KeyCode.RightArrow;
+            }
+            else if (Input.GetKey(KeyCode.UpArrow))
+            {
+                return KeyCode.UpArrow;
+            }
+            else
+            {
+                return KeyCode.DownArrow;
```

Comment:
The method "GetArrowKey" is only called when one of the four arrow keys is pressed. The other three were previously checked by a condition. Consequently, in the last case, it can only be the remaining arrow key.